### DISCUSSION TITLE
Error in license recovery

### DISCUSSION THREAD

**Discussion by MouraoL** [aclap-dev/video-downloadhelper#1238] [License & Purchase requests]
### Your question

I purchased a license on 5/18/20, but I cannot recover it by entering my email at https://www.downloadhelper.net/recover-license.
[Detalhes da transação.pdf](https://github.com/user-attachments/files/16933661/Detalhes.da.transacao.pdf)

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: pt-PT
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36
{
    "download_directory": "C:\\Users\\Mourão\\Downloads",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 5,
    "successfull_dl": 3
}

**Comment by jcfrog** [comment]
Hi, 
Sorry for that, please send us your email address at licenses@downloadhelper.net, we will send you a new license key.
